### PR TITLE
feat(version): support 18.06.0 tag format.

### PR DESCRIPTION
**Pull request by appleboy**
cc @tboerger 

Signed-off-by: Bo-Yi Wu <appleboy.tw@gmail.com>

### CODE REVIEW

**Comment by appleboy** [comment]
@tboerger See https://github.com/drone-plugins/drone-manifest/pull/20

**Comment by tboerger** [comment]
So, `docker:18.06-dind` does not provide a manifest anymore?

**Comment by appleboy** [comment]
@tboerger What do you mean? Anything I need to change?

**Comment by tboerger** [comment]
> @tboerger What do you mean? Anything I need to change?

Just because Drone fails to build.

**Comment by appleboy** [comment]
@tboerger pass

File: docker/docker/Dockerfile.linux.amd64 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM docker:18.06-dind
+FROM amd64/docker:18.06-dind
```
Comment (tboerger) [inline docker/docker/Dockerfile.linux.amd64]: I think you can keep it at `docker:18.06-dind`, that's what we are doing for all the other plugins as well. We just map the other architectures to the separate namespaces.

Comment (appleboy) [reply docker/docker/Dockerfile.linux.amd64]: OK. revered.

**Pull request merged**